gmutex/gmutex.go

```diff
@@ -504,8 +524,8 @@ func (m *Mutex) inspectObject(ctx context.Context, data io.Writer) (int, string,
 
 func (m *Mutex) url() string {
 	url := url.URL{
-		Scheme: "https",
-		Host:   "storage.googleapis.com",
+		Scheme: m.baseUrl.Scheme,
+		Host:   m.baseUrl.Host,
 		Path:   m.bucket + "/" + m.object,
```

Comment:
I didn't add support for incorporating a path prefix from the environment variable, but I assume that's ok? Happy to change if you like!